charts/lightstepsatellite/values.yaml

```diff
@@ -4,6 +4,9 @@
 
 # REQUIRED: Input your satellite key.  Satellites will not run without a valid key.  You may use either a plain text Satellite key or an existing secret with the Satellite key stored.
 
+satellite_type:
```

Comment:
I believe this is actually redundant. We're using a separate image that only deploys micro-satellites, per:

image:
  repository: lightstep/microsatellite
  version: 2021-03-22_13-16-05Z
  pullPolicy: IfNotPresent

Reply:
Sure thing, edit inbound.